StardewArchipelago/Archipelago/SlotData.cs

```diff
@@ -48,7 +48,7 @@ public class SlotData
         private const string RANDOMIZE_NPC_APPEARANCES_KEY = "randomize_appearances";
         private const string RANDOMIZE_NPC_APPEARANCES_DAILY_KEY = "randomize_appearances_daily";
         private const string MULTIWORLD_VERSION_KEY = "client_version";
-
+        private const string MOD_LIST = "mod_versions";

```

Comment:
MOD_LIST_KEY to follow the same convention as the other keys. Plus, maybe a blank space to separate from the non-key things below